apps/expo/src/app/activity.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Text, View } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+interface ActivityComponent {
+  activityName: string;
+  location?: string;
+  attending?: boolean;
+}
```

Comment:
Can you use the Activity type from the `getSchedule` endpoint for the input to the component? You can extract the activity type from the array by doing https://www.benmvp.com/blog/extracting-typescript-types-functions-objects-arrays/. You can get the output for the getSchedule endpoint from https://github.com/uoftblueprint/centre-for-dreams/blob/main/apps/expo/src/utils/api.tsx#L14

Reply:
Okay got it! Thanks :))

Reply:
Hi so actually I don't think that will work best in that case (I think idk I could be wrong). The activity component is gonna need the height and is attending parameters which arent in the type returned by schedule, also not all the items are not necessary. 

It could take in a prop with a GetScheduleType Object, is attending, and height. But that will make it a little different if we want the activity component to solely show location or is attending or nothing based on the figma. Unless we want a different Activity component for each version. (Sorry if that doesn't make sense).

Let me know if I misunderstood something tho :)). thank you!!!